### PR TITLE
styled the homepage

### PR DESCRIPTION
**Pull request by alfenn**
put Cards into a flex-wrapped container, centered the sportSelector

### CODE REVIEW

File: src/components/home-page/Home.js [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
-      <h1>Pickup Finder</h1>
-      <SportSelector sports={sports} currentSport={currentSport} setCurrentSport={setCurrentSport} />
-      {Object.entries(events).map(([eventId, event]) =>
-        <EventCard key={eventId} event={event} users={users} />
-      )}
+    <Container fluid="true" >
+      <Row>
+        <h1>Pickup Finder</h1>
+      </Row>
+      <Row className="pb-3">
```
Comment (derrickrosefan) [inline src/components/home-page/Home.js:18]: can you pass these as props to the Row component? in react-bootstrap, specifying styles via classNames is not recommended (this is a pattern seen in the regular bootstrap). if I'm wrong, let me know.

Comment (derrickrosefan) [reply src/components/home-page/Home.js:18]: (i.e. look at how style changes are passed as props for the card component https://react-bootstrap.github.io/components/cards/)

Comment (alfenn) [reply src/components/home-page/Home.js:18]: are you suggesting to use style={{paddingBottom:"10px"}} or something like that? 

isn't assigning bootstrap classes to elements the whole appeal of using bootstrap?--ig i'm curious if you have a link that explains why specifying styles via boostrap classNames is not recommended...

Comment (derrickrosefan) [reply src/components/home-page/Home.js:18]: ah, seems like padding/margins need to be specified via classNames. 
but in react-bootstrap (as opposed to vanilla bootstrap) - in general - some styles can be applied by passing the right props to the component. for example, if you want to set the background of a card to "primary" you can pass the prop bg={"primary"}.

Comment (alfenn) [reply src/components/home-page/Home.js:18]: got it that's good to know. ty!

**Pull request merged**